stacks/azure/application/centraldashboard/links-configmap.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+data:
+  links: |-
+    {
+      "menuLinks": [
+        {
+          "link": "/pipeline/",
+          "text": "Pipelines"
+        },
+        {
+          "link": "/jupyter/",
+          "text": "Notebook Servers"
+        },
+        {
+          "link": "/katib/",
+          "text": "Katib"
+        },
+        {
+          "link": "/metadata/",
```

Comment:
this has been removed.

Reply:
Thanks, I must have grabbed an outdated one because your change wasn't merged into master when I opened this.